src/lightning_habana/pytorch/strategies/parallel.py

```diff
@@ -70,8 +71,13 @@ def __init__(
         ddp_comm_wrapper: Optional[Callable] = None,
         model_averaging_period: Optional[int] = None,
         process_group_backend: Optional[str] = "hccl",
+        recipe_cache_path: Optional[str] = None,
```

Comment:
wont it be better to have the same env instead of interface changes ?

Reply:
moved to env instead of src change